azurelinuxagent/common/logcollector.py

```diff
@@ -97,28 +121,36 @@ def _expand_must_collect_files():
 
         return manifest
 
-    def _read_manifest_file(self):
-        return read_file(self.manifest_file_path).splitlines()
-
     @staticmethod
-    def _log_to_results_file(entry):
-        if isinstance(entry, list):
-            append_items_to_file(_OUTPUT_RESULTS_FILE_PATH, entry)
+    def _parametrize_filepath(path):
```

Comment:
Suggestion: Why not add macros (like `lib`, `log` and `extensionLog`) and just make the manifest relative to those.
Eg: 
```
copy,lib/provisioned -> points to /var/lib/waagent/provisioned
copy,lib/GoalState.*.xml -> points to ,/var/lib/waagent/GoalState.*.xml
copy,extensionLog/custom-script/handler.log -> points to /var/log/azure/custom-script/handler.log
```

This way we can always just replace the macros with whatever the standard in that particular VM

Reply:
Done, thanks.